docs/api/datetime.md

```diff
@@ -227,19 +231,27 @@ By default, `ion-datetime` does not show any header or title associated with the
 
 ## Buttons
```

Comment:
I'm not seeing the footer/buttons on any of the datetimes when in `md` mode, although they do show in the Stackblitz so I assume they're supposed to be there 🤔